ncm-ccm/src/main/pan/components/ccm/schema.pan

```diff
@@ -7,6 +7,7 @@ declaration template components/ccm/schema;
 
 include 'quattor/types/component';
 include 'pan/types';
+include 'components/accounts/functions';
```

Comment:
@ned21 if group does not exist, it now fallsback to root only (https://github.com/stdweird/CCM/commit/d6a4f79992ad61bb474bdd2294fa8168a545aec2), so feel free to add the ldap group to `kept_groups`